### DISCUSSION TITLE
How is the progress on Tribe?

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#1258] [Questions] [answered]
This may have already been answered on Discord (however I never use it) so I'd just want to know how far you think you are on the project or how much progress needs to be made.

Also, second question your using monkeys for the racing right? Not cars or anything else?

**Comment by Miodec** [comment, marked as answer]
Open beta is close (join the Discord to be notified) and for the racing im using tractors and wheelbarrows.


I'm of course joking, it's just bars. No monkeys, no cars.
